Components/BarProperties/index.js

```diff
@@ -22,9 +21,9 @@ export default function BarProperties({
 
   return (
     <>
-      {clickedChartType === "bar-plot" && settings.titleLabel.length > 0 && (
+      {hasCompletedStep5 === true && clickedChartType === "bar-plot" && (
```

Comment:
You could simplify the conditional check `hasCompletedStep5 === true` by simply inserting `hasCompletedStep5` directly into the condition, or?